packages/react/src/components/Segment/Segment.tsx

```diff
@@ -47,14 +51,15 @@ class Segment extends UIComponent<WithAsProp<SegmentProps>, any> {
   }
 
   renderComponent({ accessibility, ElementType, classes, unhandledProps }) {
-    const { children, content } = this.props
+    const { children, content, disabled } = this.props
 
     return (
       <ElementType
         {...rtlTextContainer.getAttributes({ forElements: [children] })}
         {...accessibility.attributes.root}
         {...unhandledProps}
         className={classes.root}
+        disabled={disabled}
```

Comment:
```
Error:(12, 8) TS2322: Type '{ disabled: string; }' is not assignable to type 'DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>'.
  Property 'disabled' does not exist on type 'DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>'.
```

`disabled` is not part of `div` props, can you check this with Reach team?

Reply:
removed, inspired by `Text` thanks